### ISSUE TITLE
Detect audio language

### ISSUE THREAD

**Issue by okan35** [nova-video-player/aos-AVP#542]
I see that in a video file where vlc detects and puts correct language names for audio, nova does not. You can see from screenshots.

Here is media data of the show
https://pastebin.com/DzBwUfxZ.
![](https://i.imgur.com/zp0ydy8.jpg) ![](https://i.imgur.com/3Eow8Ey.jpg)

**Comment by Pentaphon** [comment]
I agree with auto track selection being a long term goal. Nova should someday be able to read the track tags and select them for the user according to the settings.

**Comment by Tyfomir** [comment]
This will undoubtedly be a very useful feature.

**Comment by Pentaphon** [comment]
@courville I think track language detection is a good v7 milestone. What do you think? Nova needs to show more info and autoselect it for the user, especially as videos are using more audio and subtitle tracks than ever before.

**Comment by courville** [comment]
avos multimedia engine now reports audio stream title/language metadata that can be redundant with codec information displayed in the UI.
